src/main/menus/file.js

```diff
@@ -3,31 +3,62 @@ import * as actions from '../actions/file'
 import { userSetting } from '../actions/marktext'
 import userPreference from '../preference'
 
-const { autoSave } = userPreference.getAll()
-const notOsx = process.platform !== 'darwin'
+export default function (recentlyUsedFiles) {
+  const { autoSave } = userPreference.getAll()
+  const notOsx = process.platform !== 'darwin'
+  let fileMenu = {
+    label: 'File',
+    submenu: [{
+      label: 'New File',
+      accelerator: 'CmdOrCtrl+N',
+      click (menuItem, browserWindow) {
+        actions.newFile()
+      }
+    }, {
+      label: 'Open...',
+      accelerator: 'CmdOrCtrl+O',
+      click (menuItem, browserWindow) {
+        actions.open(browserWindow)
+      }
+    }]
+  }
 
-export default {
-  label: 'File',
-  submenu: [{
-    label: 'New File',
-    accelerator: 'CmdOrCtrl+N',
-    click (menuItem, browserWindow) {
-      actions.newFile()
+  if (notOsx) {
+    let recentlyUsedMenu = {
+      label: 'Open Recent',
+      submenu: [{
+        label: 'Clear Recently Used',
```

Comment:
Please put the `Clear Recently Used` at the bottom of menu.(it means the last menu item).